src/components/tabs/themes/light/tabs.base.scss

```diff
@@ -4,7 +4,10 @@
 $tabs-animation-function: cubic-bezier(.35, 0, .25, 1);
 
 :host {
-    display: block;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    width: 100%;
```

Comment:
Using height/width: 100% looks like a problem waiting to happen.
This should be decision of the owner of Tabs component.

Reply:
Since the issue those not specify if the fix should be only for tabs that are inside a `flex` container, using `height:100%` and `width:100%` ensures that the fix will work even if the tabs are not inside a `flex` container. If that is not a requirement I can do it with `flex-grow: 1`;

Reply:
What are your concerns, do you see a use case where this can have harmful effect?

Reply:
- Width: you already declared tabs to be "flex", that's block by default so width: 100% is implicit.
- 2 examples of harmful effects:
  1. when in a flex layout it changes the auto basis, so if you want to put a tabs and a div side by side, the tabs will use all the space, e.g.
     
    ![image](https://user-images.githubusercontent.com/12942794/227935293-d79207d2-abfe-4fc4-a7d4-e2224c184b3d.png)

  2. when in a regular layout, if you put a tabs and a button side by side it will make the whole layout overflow
     e.g.
     ![image](https://user-images.githubusercontent.com/12942794/227936179-7fd682dd-2d6b-4145-b631-80e0838816ab.png)

Why having different styling for WC and Angular? After all the refinement Angular has gone through, we start from scratch only to repeat the same mistakes or different ones?